apps/dialtone-documentation/docs/components/toast.md

```diff
@@ -90,6 +90,62 @@ vueCode='
 '
 showHtmlWarning />
 
+### With duration
+
+It's recommended to use a time of at least 6000 ms (minimum duration validated in the component) to give users enough time to read the toast. Take into account that the time necessary to read and comprehend the message could vary in users. For instance, users using assistive technology, or users with language barriers could potentially need more time to read and understand the message.
+If the duration is not provided the toast won't disappear automatically.
+
+<code-well-header>
+  <dt-button @click="toggleDurationToast(true)">Toggle Example</dt-button>
+  <example-toast
+    class="d-zi-notification"
+    :show.sync="showDurationToast"
```

Comment:
sync is not valid in vue3, we can just use v-model instead or put some sort of comment explaining the difference to avoid confusion